irctest/server_tests/utf8.py

```diff
@@ -53,15 +53,16 @@ def testNonutf8Realname(self):
             raise runner.IsupportTokenNotSupported("UTF8ONLY")
 
         self.addClient()
-        self.sendLine(2, "NICK foo")
-        self.clients[2].conn.sendall(b"USER username * * :i\xe8rc\xe9\r\n")
-
-        d = self.clients[2].conn.recv(1024)
-        if b" FAIL " in d or b" 468 " in d:  # ERR_INVALIDUSERNAME
+        self.sendLine(2, "NICK bar")
+        self.sendLine(2, b"USER username * * :i\xe8rc\xe9")
+        m = self.getRegistrationMessage(2)
```

Comment:
This will crash if the server returned the non-UTF8 username.

Reply:
Ah okay I see, will revert that portion.